### ISSUE TITLE
IMixinConfigPlugin acceptTargets not working: UnmodifiableCollection does not support remove/removeAll

### ISSUE THREAD

**Issue by PG85** [SpongePowered/Mixin#545]
Using IMixinPluginConfig with forge 1.16.5 / org.spongepowered:mixin:0.8:processor

https://github.com/SpongePowered/Mixin/blob/master/src/main/java/org/spongepowered/asm/mixin/extensibility/IMixinConfigPlugin.java#L91

States that entries can be removed from Set<String> myTargets, however it is unmodifiable so removeAll and remove throw an UnsupportedOperationException.
```
java.lang.UnsupportedOperationException at java.util.Collections$UnmodifiableCollection.removeAll(Collections.java:1068)
```

**Comment by PG85** [comment]
Workaround is to use reflection to grab the wrapped set, which can be modified:

```
Field field = myTargets.getClass().getSuperclass().getDeclaredField("c");
field.setAccessible(true);
Collection<String> wrappedSet = (Collection<String>)field.get(myTargets);
```

**Comment by LlamaLad7** [comment]
This was fixed in Mixin 0.8.3.

**Comment by Mumfrey** [comment]
Duplicate of #485 which was fixed in 65b4f9d55a1eb9f5c5491e8a1cfcee3ba9b8fb0e. Please ensure you're using the latest version when reporting issues.

**Comment by PG85** [comment]
That explains it, thanks! In case it's useful to anyone, Minecraft Forge 1.16.5-36.2.20 apparently updated to Mixin 0.8.4 (5 days ago).
